connect-inject/container_init_test.go

```diff
@@ -79,6 +101,8 @@ func TestHandlerContainerInit(t *testing.T) {
 			container, err := h.containerInit(tt.Pod(minimal()))
 			require.NoError(err)
 			actual := strings.Join(container.Command, " ")
+			//fmt.Println(actual)
```

Comment:
Hi sorry I forgot about this, I have removed the statements and added an update to the readme to mention go 1.11.4+ which introduced go.sum changes because of symlinks.